src/lib/bridge/handlers/fetch_bluegem.ts

```diff
@@ -0,0 +1,44 @@
+import {SimpleHandler} from './main';
+import {RequestType} from './types';
+
+export interface BluegemPatternData {
+    playside_blue: number;
+    playside_purple?: number;
+    playside_gold?: number;
+    backside_blue?: number;
+    backside_purple?: number;
+    backside_gold?: number;
+    playside_contour_blue?: number;
+    playside_contour_purple?: number;
+    backside_contour_blue?: number;
+    backside_contour_purple?: number;
+}
+
+export interface FetchBluegemRequest {
+    type: string;
```

Comment:
Perhaps we should let them pass the `iteminfo` instead of the type + paintseed, so that we don't need to do string replacement in multiple places.